style.css

```diff
@@ -539,35 +540,41 @@ section {
 
 .popup {
   position: fixed;
-  top: 50%;
+  top: 49%;
   left: 50%;
   transform: translate(-50%, -50%);
   background-color: #fff;
   padding: 22px;
-  border-radius: 8px;
+  border-radius: 16px;
+  border: 1px solid lightgray;
   box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
   z-index: 9999;
-  width: 80vw;
+  width: 16rem;
+  height: 31.4rem;
   display: flex;
 }
```

Comment:
Great job on using JavaScript to populate your projects! I just wanted to give you some friendly styling suggestions to help your project look even closer to the Figma template and appear more professional. 

- [ ] In the mobile view, I noticed that the popup seems a bit small, especially on larger screens like the iPhone XR. 📱 I suggest working with percentages to set the size of your popup, which will ensure it looks great across different mobile devices. 📏

- [ ] Additionally, please take a look at the small elements that contain HTML, CSS, and JavaScript. They don't appear aligned and centered at the moment. 😕 It would be fantastic if you could make some adjustments to achieve better alignment and centering. ✨

- [ ] Lastly, there are a few spacing issues I noticed with CANOPY, Backe end dev, and 2015. It would be awesome if you could double-check and fix the spacing for these elements. 😊

   ### Your design:
   ![image](https://github.com/josman227/portfolio.github.io/assets/107952724/de71786b-fdcc-492b-82d7-a9e14da87ca9)

   ### Figma design:
   ![image](https://github.com/josman227/portfolio.github.io/assets/107952724/b6fc114c-8644-4e7c-8ff3-f08a009aa0a2)